src/main/scala/akka/contrib/process/BlockingProcess.scala

```diff
@@ -133,20 +142,19 @@ class BlockingProcess(
     val blockingIODispatcherId = context.system.settings.config.getString(BlockingIODispatcherId)
 
     try {
-      val selfRef = context.self
       val selfDispatcherAttribute = ActorAttributes.dispatcher(blockingIODispatcherId)
 
       val stdin = StreamConverters.fromOutputStream(() => process.getOutputStream())
         .withAttributes(selfDispatcherAttribute)
-        .mapMaterializedValue(_.andThen { case _ => selfRef ! StdinTerminated })
+        .mapMaterializedValue(_.andThen { case _ => self ! StdinTerminated })
```

Comment:
@longshorej - we are enclosing over an `ActorRef` within an async context. I know that `self` should refer back to self reference, but won't it be safer to rely on the `ActorRef` created outside of this async context (i.e. `selfRef`)?